monitoring/slack_alerts/auth0_log_stream_alert/src/auth0_log_stream_alert.py

```diff
@@ -143,6 +145,33 @@ def should_alert_for_event(log_event):
     return True
 
 
+def should_log_description_for_event(log_event):
+    """
+    Should the Slack alert for this event include the description?
+
+    We can't include the description in the general case, because it might
+    contain PII we don't want in Slack, e.g.
+
+        User jo@example.com failed to log in successfully
+
+    But in certain cases it may be useful to log a description if
+    we know it's safe.
+    """
+    # These keys should match those set in the event transform rule
+    # https://github.com/wellcomecollection/identity/blob/main/infra/scoped/auth0-logs.tf#L55
+    log_event_type = log_event["log_event_type"]
+    log_description = log_event["log_description"]
+
+    # If we're hitting API rate limits, we can log the name of the
+    # endpoint that's being limited without giving anything away.
+    if log_event_type == "api_limit" and re.match(
```

Comment:
I'd prefer this, but only marginally
```suggestion
    return log_event_type == "api_limit" and re.match(
```

Reply:
I'm going to leave this as-is; the intention was that over time we might add other `if` branches for cases where it's safe to let the description through.